CUJExecutor/yesProfileCuj

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+#Copyright 2020 Google LLC
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+
+INPUT_FILE=${1:-input.txt}
+FILE_NAME="${INPUT_FILE%.*}"
+PRE_CUJ=$(sed '1q;d' $INPUT_FILE)
+POST_CUJ=$(sed '2q;d' $INPUT_FILE)
+RECORD=$(sed '4q;d' $INPUT_FILE)
+
+
+
+##FORMAT PRE AND POS CUJS
+PRE_CUJ=${PRE_CUJ//\‘/\'}
+PRE_CUJ=${PRE_CUJ//\’/\'}
+POST_CUJ=${POST_CUJ//\‘/\'}
+POST_CUJ=${POST_CUJ//\’/\'}
+
+#use escape char to allow special characters (which would terminate the argument in the am instrument command or do other undesirable things).
+SpecialChars=(";"  "'" "&" "%" "(" ")" "^" "#" "@" "!" "+")
+for char in ${SpecialChars[@]}; do
+	exp="\\${char}"
+	PRE_CUJ=${PRE_CUJ//$exp/\\$char}
+	POST_CUJ=${POST_CUJ//$exp/\\$char}
+done
+# Wrap strings in \" to allow for spaces within them. 
+PRE_CUJ="\\\"$PRE_CUJ\\\""
+POST_CUJ="\\\"$POST_CUJ\\\""
+
+
+echo "Resetting the profile...."
+adb shell cmd package compile --reset com.google.android.apps.maps
+
+#Get the entire CUJ into the profile:
+echo "Walking through the CUJ to (hopefully) add it to the profile...."
+#echo $PRE_CUJ > walk_through_cuj.txt
+#echo "$POST_CUJ" >> walk_through_cuj.txt
+#echo "0" >> walk_through_cuj.txt
+
+./gradlew installDebugAndroidTest
```

Comment:
The `noProfileCuj` script didn't have this.

Reply:
yeah its a bit messy, but this is because noProfileCuj calls executeCuj, which builds